CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 + The keys on disk are now stored in the `~/.notary/private` directory, rather than in a key hierarchy that separates them by GUN and by role.  Notary will automatically migrate old-style directory layouts to the new style.  **This is not forwards-compatible against notary<0.4.2 and docker<=1.12** [#872](https://github.com/docker/notary/pull/872)
 + A new changefeed API has been added to Notary Server. It is only supported when using one of the relational database backends: MySQL, PostgreSQL, or SQLite.[#1019](https://github.com/docker/notary/pull/1019)
 
+## [v0.4.3](https://github.com/docker/notary/releases/tag/v0.4.3) 1/3/2017
```

Comment:
Sorry, I think I merged in master to pull in current as per my usual development cycle. Will fix, same for all other duplicated changes